### PR TITLE
Update one-cmds script for aarch64 support

### PR DESCRIPTION
**Pull request by lemmaa**
- intruduce one-prepare-venv.u1804.aarch64

ONE-DCO-1.0-Signed-off-by: Sung-Jae Lee <sj925.lee@samsung.com>

for: #11036 
draft: #10564

### CODE REVIEW

File: compiler/one-cmds/one-prepare-venv.u1804.aarch64 [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```
Comment (seanshpark) [inline compiler/one-cmds/one-prepare-venv.u1804.aarch64:1]: As I remember, this file will be copied as `one-prepare-venv.u1804` with
`get_filename_component(ONE_COMMNAD_FILE_NAME ${ONE_COMMAND} NAME_WE)` line.
Can you check again with copied file name?

FYI, I proposed to use `one-prepare-venv.u1804-aarch64`

Comment (seanshpark) [reply compiler/one-cmds/one-prepare-venv.u1804.aarch64:1]: My local build shows this is OK.

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/one-cmds/one-prepare-venv.u1804.aarch64 (hunk 1 above)
Comment (lemmaa) [reply compiler/one-cmds/one-prepare-venv.u1804.aarch64:1]: > FYI, I proposed to use `one-prepare-venv.u1804-aarch64`

Oops, my mistake.

**Pull request merged**